From: Gleb Natapov <gleb@redhat.com>
Date: Tue, 15 Jun 2010 08:11:28 -0300
Subject: [PATCH] do not advertise hpet to a guest OS.


Signed-off-by: Gleb Natapov <gleb@redhat.com>
---
 v1->v2:
  - do not remove code to generate HPET tables, only call to the
    function.

--
			Gleb.

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 src/acpi.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/src/acpi.c b/src/acpi.c
index 244536a..1962632 100644
--- a/src/acpi.c
+++ b/src/acpi.c
@@ -635,7 +635,6 @@ acpi_bios_init(void)
     ACPI_INIT_TABLE(build_fadt(bdf));
     ACPI_INIT_TABLE(build_ssdt());
     ACPI_INIT_TABLE(build_madt());
-    ACPI_INIT_TABLE(build_hpet());
     ACPI_INIT_TABLE(build_srat());
 
     u16 i, external_tables = qemu_cfg_acpi_additional_tables();
-- 
1.7.0.3

